src/main/java/org/jenkinsci/test/acceptance/po/Job.java

```diff
@@ -408,4 +408,15 @@ public ScmPolling pollScm() {
 
         return links;
     }
+
+    /**
+     * Deletes the current job
+     */
+    public void delete() {
+        this.open();
+        clickLink("Delete Project");
```

Comment:
@andresrc @kwhetstone I agree with @olivergondza here, I was in a similar situation where a `FreeStyleMultiBranchProject` changed the usual UI and `Job.addShellStep()' were not working, overriding it in  `FreeStyleMultiBranchProject` was very easy